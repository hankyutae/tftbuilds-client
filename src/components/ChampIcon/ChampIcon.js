import React from 'react';
import './ChampIcon.css';
//import ImgLink from '../../services/create-img-link-service';

export default class ChampionIcon extends React.Component{
  render(){
    return(
      <div className='champ-icon-general'>

      </div>

    );
  }
}