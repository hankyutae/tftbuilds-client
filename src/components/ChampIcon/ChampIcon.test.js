import React from 'react';
import ReactDOM from 'react-dom';
import ChampIcon from './ChampIcon';
import renderer from "react-test-renderer";

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
      <ChampIcon/>
  , div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders the UI as expected", () => {
  const tree = renderer
    .create( 
      <ChampIcon/>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});