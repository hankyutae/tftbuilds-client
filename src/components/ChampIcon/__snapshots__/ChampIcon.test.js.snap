// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="champ-icon-general"
/>
`;
